test/functional/Valhalla/src/org/openj9/test/lworld/ValhallaAttributeTests.java

```diff
@@ -235,7 +235,7 @@ static value class ImplicitClass {
 	/* Test to verify JVM_IsImplicitlyConstructibleClass. */
 	@Test
 	public static void testValueClassIsImplicitlyConstructible() {
-		ImplicitClass ic = (ImplicitClass)jdk.internal.value.ValueClass.zeroInstance(ImplicitClass.class);
+		Assert.assertTrue(jdk.internal.value.ValueClass.isImplicitlyConstructible(ImplicitClass.class));
```

Comment:
nit: indentation looks off for the changes in this file.

Reply:
It is fine in my desktop editor. It seems github web interface thing.